contexts/chat.tsx

```diff
@@ -31,8 +31,8 @@ interface ChatContextState {
   setSubTool: React.Dispatch<React.SetStateAction<string>>;
   setScript: React.Dispatch<React.SetStateAction<string>>;
   setScriptContent: React.Dispatch<React.SetStateAction<Block[] | null>>;
-  tool: Tool;
-  setTool: React.Dispatch<React.SetStateAction<Tool>>;
+  rootTool: Tool;
```

Comment:
I keep getting confused that this is only the "root tool" and not the whole tool, so I renamed it. Happy to switch it back if folks disagree with this change.

Reply:
I like this